pids.md

```diff
@@ -7,6 +7,6 @@ permalink: /pids/
 {% for vid in vids %}
   {% assign urlparts = vid.url|split:"/" %}
   {% assign vidno = urlparts[1] %}
-  <h3><a href="{{vid.url}}">0x{{urlparts[1]}}</a></h3>
+  <h3><a href="{{vid.url}}">0x{{vidno}}</a></h3>
```

Comment:
Hm. I suspect it's the empty lines from the assign statements, which markdown is treating as empty paragraphs. Maybe [put it inside a div](http://stackoverflow.com/questions/701042/disable-markdown-for-a-block) or try moving them all onto one line?